README.md

```diff
@@ -20,6 +20,7 @@ Set the supported components to monitor and the tunings like number of iteration
 cerberus:
     kubeconfig_path: ~/.kube/config                      # Path to kubeconfig
     watch_nodes: True                                    # Set to True for the cerberus to monitor the cluster nodes
+    watch_operators: True                                # Set to True for cerberus to monitor cluster operators. Parameter is optional, will set to false if not specified
```

Comment:
Maybe rename watch_operators to watch_cluster_operators to make it explicit to the user that enabling this will just monitor the cluster operators.